java/opendcs/src/main/java/decodes/tsdb/comprungui/CompRunGuiFrame.java

```diff
@@ -1074,6 +1131,8 @@ private void runSelectedComps(Collection<DbComputation> compVector, Logger origi
 			@Override
 			public List<CTimeSeries> doInBackground()
 			{
+				myinputs.clear();
```

Comment:
This should probably be around line 1129 with the myoutputs.clear(). Just for consistency.